### PR TITLE
Add Events in the Netherlands

### PR DESCRIPTION
**Pull request by davidvanleeuwen**
**[https://github.com/awkward/awesome-netherlands-events](https://github.com/awkward/awesome-netherlands-events)**

**This list contains awesome (tech related) events happening (and are mostly recurring) in The Netherlands like: conferences, meetups and parties 🎉**

# By submitting this pull request I confirm I've read and complied with the below requirements.

Failure to properly do so will just result in the pull request being closed and everyone's time wasted. Please read it twice. Most people miss many things.

- I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title. For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should include a short description about the list project/theme (not the list itself), and should be added at the bottom of the appropriate category.
- The list I added
	- **has been around for at least 30 days,**
	- is the result of hard work and is the best I could possibly produce,
	- is a non-generated Markdown file in a GitHub repo,
	- is not a duplicate,
	- includes a succinct description of the project/theme at the top ([example](https://github.com/willempienaar/awesome-quantified-self)),
	- includes a project logo/illustration (whenever possible) at the top right of the readme ([example](https://github.com/sindresorhus/awesome-electron)). (The logo should link to the project website or any relevant website. The logo should be high-DPI, so set it to maximum half the width of the original image.),
	- only has awesome items **with descriptions** *(Awesome lists are curations of the best, not of everything)*,
	- contains the [awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge) on the right side of the list heading,
	- has a Table of Contents section **named `Contents`** as the first section,
	- has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license) *(Meaning something like CC0, NOT a code licence like MIT, BSD, Apache, etc)* *(If you use a license badge, it should be SVG, not PNG)*,
	- has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines),
	- doesn't include a Travis badge,
	- has been checked for inconsistent formatting, spelling, and grammar:
		- Each link description starts with an uppercase character and ends with a period.
		- Drop all the `A` / `An` prefixes in the descriptions.
		- Consistent naming. For example, `Node.js`, not `NodeJS` or `node.js`.

### CODE REVIEW

**Comment by legodude17** [comment]
I would suggest adding `in The Netherlands` to the heading.

**Comment by sindresorhus** [comment]
Should be:

> Events in the Netherlands

**Comment by sindresorhus** [comment]
The repo title is too vague. Should be either `awesome-events-netherlands` or `awesome-netherlands-events`.

File: readme.md [hunk 1]
```diff
@@ -511,6 +511,7 @@
 - [Free Software](https://github.com/johnjago/awesome-free-software) - Free as in freedom.
 - [Framer](https://github.com/podo/awesome-framer) - Prototyping interactive UI designs.
 - [Markdown](https://github.com/BubuAnabelas/awesome-markdown) - Markup language.
+- [Events (🇳🇱)](https://github.com/awkward/awesome-events) - Tech related events in The Netherlands like: conferences, meetups and parties.
```
Comment (sindresorhus) [inline readme.md]: Too long description. Try:

> Tech-related events in the Netherlands.

**Comment by sindresorhus** [comment]
I would also include a link to http://lanyrd.com/places/netherlands/ to track all conferences.

And maybe also https://www.meetup.com/find/?allMeetups=false&keywords=tech&radius=100&userFreeform=netherlands&gcResults=Netherlands%3ANL%3Anull%3Anull%3Anull%3Anull%3Anull%3A52.132633%3A5.2912659999999505&change=yes&sort=member_count

**Comment by davidvanleeuwen** [comment]
Thanks for the feedback 👍  Made some changes ^

**Pull request merged**